README.md

```diff
@@ -1,91 +1,189 @@
 # CSS Media Queries
 
-## Overview
-In this lesson we will look at using CSS3 Media Queries to detect viewport sizes and adjust CSS styles accordingly.
+## Problem Statement
+
+A component of designing and building responsive layouts is being mindful
+of the wide variety of screen sizes. Originally, media queries could be used
+to set specific breakpoints based on a specific set of devices. As the market
+of web-connected electronics has grown, media queries have evolved to be more 
+inclusive of new sizes and aspect ratios. What are media queries, exactly, and 
+how can they be taken advantage of in building page layouts?
 
 ## Objectives
-1. The purpose of media queries.
-2. Media Query syntax.
-3. Accepted Devices.
-4. Min and max conditions.
-5. Choosing break point sizes.
 
-## Introducing Media Queries
-<iframe width="640" height="480" src="//www.youtube.com/embed/MYOT5FDG8gk?rel=0&modestbranding=1" frameborder="0" allowfullscreen></iframe>
+1. What is the purpose of media queries
+2. What is the Media Query syntax
+3. Where should break points be created
```

Comment:
I think it's appropriate to teach the word "break point" be consistent in spacing and make teaching the word an objective.